src/utils/saml.ts

```diff
@@ -469,6 +470,272 @@ export const getAuthorizeRequestTamperer = (
 //  Validate response
 //
 
+const utcStringToDate = (value: string, tag: string): Date => {
+  const maybeDate =
+    value.length === 24
```

Comment:
what's happening here ? (probably worth's a comment)

Reply:
Totally removed, `UTCISODateFromString` decoder supports both the UTC strings type (with and without milliseconds)